### DISCUSSION TITLE
3주차 반성문

### DISCUSSION THREAD

**Discussion by llddang** [BDD-CLUB/challenge-every-week-posting#78] [6. 반성문]
오늘 자고 일어나서 블로그 작성해야지 생각하고 있었는뎅 자고있는 사리에 일주일이 지났네용!
저는 오늘이 일요일이라서 착각했습니다!!
일요일 ~ 토요일이 일주일이라고 머리에 박겠습니다!
죄송합니다!
열심히 블로그 작성하겠습니다!

**Comment by shkisme** [comment]
헐 요일 헷갈릴 만 하네요... 화이팅입니다~!
